BlueSlug/wecount.inclusivedesign.ca
# feat: Implement the resource detail page

## Description

Implement the resource detail page. This PR also fixes a bug that newly created resources break the search and filter functions on the resources page.

## Steps to test

1. Go to "Resouces" page;
2. Clicking a resource leads to a resource detail page. The page style should match its design on [Figma](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=3840%3A1463);
3. Test the page with UIO. The page should respond correctly;
4. Go to the admin site -> resources -> click on a resource;
5. The resource preview should display and has the same style as the resource detail page;
6. Create a new resource and save -> go to the resources page -> perform search and/or filter -> search/filter should complete with correct results.

**Expected behavior:** 

See above.
